.github/workflows/deploy/action.yaml

```diff
@@ -0,0 +1,25 @@
+inputs:
+  role: 
```

Comment:
Secrets aren't available in the composite action - so we need to pass them in directly